cmd/session.go

```diff
@@ -354,3 +327,69 @@ func SessionTerminate(cmd *cobra.Command, args []string) error {
 	ui.Successf("Session terminated")
 	return nil
 }
+
+// sessionSelectSSHExecRef selects an exec id from all sessions in the Unweave environment or whether to create a new
+// provides an option to create a new Exec an error or exits if unrecoverable
+func sessionSelectSSHExecRef(cmd *cobra.Command, execRef string, allowNew bool) (string, bool, error) {
```

Comment:
Could you name the return variables so it's clear what the bool and string are?

Reply:
No worries, will adopt this style through the codebase.

Reply:
No you don't need to necessarily. Just when it's not clear what the return value is. Alternatively you could add it to the doc string.

Reply:
Great, thanks for clarifying!

Reply:
Implemented